src/icesat2_tracks/config/IceSAT2_startup.py

```diff
@@ -15,42 +15,27 @@
 from matplotlib.gridspec import GridSpec
 import matplotlib.colors as colors
 import pandas as pd
-
+from icesat2_tracks.local_modules import m_colormanager_ph3 as M_color
+from icesat2_tracks.local_modules import m_tools_ph3 as MT
+from icesat2_tracks.local_modules import m_general_ph3 as M
 
 import string
 
 import xarray as xr
-#xr.set_options(display_width=80, display_style='text')
-
-
-import sys
-import imp
-
-
-
-# my own libraries:
-#import m_general as M
+xr.set_options(display_width=80, display_style='text')
 
-#import AA_plot_base as AA
-def json_load(name, path, verbose=False):
-    import json
-    full_name= (os.path.join(path,name+ '.json'))
+## Read folders and configuration paths
+config_dir_path = os.path.dirname(__file__)
+mconfig=MT.json_load('config',config_dir_path)
 
-    with open(full_name, 'r') as ifile:
-        data=json.load(ifile)
-    if verbose:
-        print('loaded from: ',full_name)
-    return data
+## check folders exist. Create if dont.
+for folder_name, folder_path  in mconfig["paths"].items():
+    full_path = os.path.abspath(folder_path)
+    if not os.path.exists(full_path): 
+          os.mkdir(full_path)
```

Comment:
You could make this shorter and more robust: 
```suggestion
    pathlib.Path(full_path).mkdir(exist_ok=True)
```